client/domain/CodeEvalResultObjectFactorySpec.js

```diff
@@ -22,6 +22,7 @@ describe('CodeEvalResultObjectFactory', function() {
   var CODE = "code";
   var OUTPUT = "output";
   var CORRECTNESS_TEST_RESULTS = "correctnessTestResults";
+  var LAST_TASK_RESULTS = "s";
```

Comment:
I still don't understand this. Why is it "s"? I thought we talked previously about making the test result variables lists instead of strings.